src/bundle/Resources/public/css/button.css

```diff
@@ -0,0 +1,83 @@
+.ez-button {
+    display: inline-block;
+    line-height: normal;
+    white-space: nowrap;
+    vertical-align: middle;
+    text-align: center;
+    cursor: pointer;
+    -webkit-user-drag: none;
+    user-select: none;
+    box-sizing: border-box;
+
+    text-decoration: none;
+    padding: 0.5em 1em;
```

Comment:
@inakijv `.ez-button` is global but I get why there's such a difference. In the UI guide lines, you use bootstrap.css which defines `html {font-size: 10px}` so in the end, 0.5em/1em here is almost the same of .85rem/1.75rem (1 or 2px of difference :))